mitodl/open-discussions
# Add embedded youtube videos as course resources

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #3156 

#### What's this PR do?
- Parses the `course_embedded_media` section of OCW master JSON files and ETL's any Youtube videos found into `ContentFile` objects.

#### How should this be manually tested?
- Run the new migration
- Run `manage.py backpopulate_ocw_files`

When complete, there should be new `ContentFile` objects with `content_type=video`, `file_type=video/youtube` and a non-null `image_src` field.  Most of these should have a non-null `content` field as well.
